README.md

```diff
@@ -1,160 +1,38 @@
 # Zenith
 
-Zenith substitutes PostgreSQL storage layer and redistributes data across a cluster of nodes
+Zenith substitutes PostgreSQL storage layer and redistributes data across a cluster of nodes.
 
-## Running local installation
+## Architecrure overview
 
-1. Install build dependencies and other useful packages
+A Zenith installation consists of Compute nodes, Storage engine and .
 
```

Comment:
For me it kinda helpful to have an installation overview in the base README.md. When you see a new project it helps a lot to start hacking